python/remote_api/src/lynx/kite.py

```diff
@@ -1161,7 +1145,7 @@ def _fn_id(fn: Callable):
 
 
 def _is_lambda(f: Callable):
-  def LAMBDA(): return 0
+  LAMBDA = lambda: 0 # fmt: off (autopep8 wants to turn this into a function.)
```

Comment:
So the original author (@erbenpeter) wrote this correctly, but the pre-commit hook turned it into the silly version automatically. :smile: